ui/src/config/section/infra/primaryStorages.js

```diff
@@ -133,6 +133,24 @@ export default {
       dataView: true,
       show: (record) => { return ['Maintenance', 'PrepareForMaintenance', 'ErrorInMaintenance'].includes(record.state) }
     },
+    {
+      api: 'changeStoragePoolScope',
+      icon: 'arrow-right-outlined',
+      label: 'label.action.change.primary.storage.scope',
+      dataView: true,
+      popup: true,
+      show: (record) => {
+        return (record.state === 'Disabled' &&
+          (record.hypervisor === 'KVM' ||
+           record.hypervisor === 'VMware' ||
+           record.hypervisor === 'HyperV' ||
+           record.hypervisor === 'LXC' ||
+           record.hypervisor === 'Any' ||
+           record.hypervisor === 'Simulator')
```

Comment:
```suggestion
          ['KVM', 'VMware', 'HyperV', 'LXC', 'Any', 'Simulator'].includes(record.hypervisor)
```

Reply:
PR description says feature is supported for KVM, VMware and Simulator only?

Reply:
I have updated the PR description.
This feature is tested and supported for a limited no. of storage-hypervisor configurations.
But we are not gating this functionality for other combinations.

Hypervisors that are given here is same as the hypervisors which support zone scope primary storage.

Reply:
Then shouldn't we show the action only for the supported hypervisors? More hypervisors can be added once we have support for them?

Reply:
Only two hypervisors are tested.
But we will allow root admin the ability to use the feature for other hypervisors as well.